Settlement/Transaction.ts

```diff
@@ -80,11 +82,11 @@ export namespace Transaction {
 
 	export function toCsv(transactions: Transaction[]): string {
 		let result =
-			"authorization,reference,type,card,scheme,area,created,currency,gross,fee,net,reserve amount,reserve payout\r\n"
+			"authorization,reference,type,NNSS,card,scheme,area,created,currency,gross,fee,net,reserve amount,reserve payout\r\n"
 		for (const value of transactions)
-			result += `"${value.authorization}","${value.reference}","${value.type}","${value.card}","${value.scheme}","${
-				value.area
-			}","${value.created}","${value.currency}","${value.gross}","${
+			result += `"${value.authorization}","${value.reference}","${value.type}","${value.nnss ?? "N"}","${
```

Comment:
I think its nicer for the people who are reading the csc-files if the `nnss` is set to "Y" or nothing. And when we fetch the BV files there are no "N" set, so maybe just leave nnss empty.
```suggestion
			result += `"${value.authorization}","${value.reference}","${value.type}","${value.nnss ?? ""}","${
```

Reply:
So why do we use the type `nnss?: "Y" | "N"` ?

Reply:
So why do we use the type `nnss?: "Y" | "N"` ?